README.md

```diff
@@ -67,38 +73,64 @@ plugins:
 custom:
   vpcDiscovery:
     vpcName: '${opt:env}'
-    subnetNames: # optional if securityGroupNames are specified
-      - '${opt:env}_<name of subnet>'
-    securityGroupNames: # optional if subnetNames are specified
-      - '${opt:env}_<name of security group>'
+    # optional if `securityGroups` option is specified
+    # list of tag key and values 
+    subnets:
+      # tag key
+      - tagKey: <tag_name>
+        # an array of values
+        tagValues:
+          - '<tag_value>'
+    # optional if `subnets` option is specified
+    # list of tag key and value or names
+    securityGroups:
+      # tag key
+      - tagKey: <tag_name>
+        # an array of values
+        tagValues:
+          - '<tag_value>'
+      # optional if `tagKey` and `tagValues` are specified
+      - names:
+        - '<security_group_name>'
 
-# (Optional) set a config for the specific function
+# Optional: Either set `custom.vpcDiscovery` or `functions.<function name>.vpcDiscovery`
 functions:
-  example1:
+  example:
     handler: handler.example
-    # inherit `custom.vpcDiscovery` config if specified
+    # inherit `custom.vpcDiscovery` config in case `custom.vpcDiscovery` is specified
```

Comment:
Nitpick: I realized the position of this comment is a bit confusing. Is it referring to the `example` function or the one below it. If it's for the `example` function then it probably should go above the function. Maybe some new lines in between each functions could also clear it up